sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileCopyOptions.cs

```diff
@@ -27,6 +27,11 @@ public class ShareFileCopyOptions
         /// </summary>
         public string FilePermission { get; set; }
 
+        /// <summary>
+        /// Format of File Permission.
```

Comment:
Update the description to be more than what's in the name.

"Specifies the format in which the file permission is returned. If unspecified or explicitly set to SDDL, the permission is returned in SDDL format. If explicitly set to binary, the permission is returned as a base64 string representing the binary encoding of the permission."

Or this can be added to the description of `FilePermissionFormat` and you can reference from there.

Reply:
Fixed, good call out!